app/controllers/webhooks_controller.rb

```diff
@@ -0,0 +1,54 @@
+class WebhooksController < ApplicationController
+
+  skip_before_action :verify_authenticity_token
+  before_action :verify_request
+
+  def receive
+    org_uid = params[:org_uid]
+    organization = Maestrano::Connector::Rails::Organization.find_by_uid(org_uid)
+    if organization
+      Rails.logger.debug("WebhooksController.receive with params: #{webhook_params}")
+      entities_hash ={params[:entity].singularize.capitalize => [webhook_params]}
+      Maestrano::Connector::Rails::PushToConnecJob.perform_later organization, entities_hash
+    else
+      Rails.logger.debug('WebhooksController.receive: could not find organization: ' + org_uid)
+    end
+    head 200, content_type: 'application/json'
+  end
+
+  private
```

Comment:
Duplicated 'private' statement (and your private methods should have one more level of indentation)